recipes/ssl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "ssl" %}
```

Comment:
Let's call it `backports.ssl`. There is an `ssl` package for Python too. So this is a bit confusing if we name it as is.